README.md

```diff
@@ -6,9 +6,9 @@ Copyright (c) ONNX Project Contributors
 
 <p align="center"><img width="40%" src="https://github.com/onnx/onnx/raw/main/docs/onnx-horizontal-color.png" /></p>
 
-[![Build Status](https://dev.azure.com/onnx-pipelines/onnx/_apis/build/status/Windows-CI?branchName=main&label=Windows)](https://dev.azure.com/onnx-pipelines/onnx/_build/latest?definitionId=5&branchName=main)
-[![Build Status](https://dev.azure.com/onnx-pipelines/onnx/_apis/build/status/Linux-CI?branchName=main&label=Linux)](https://dev.azure.com/onnx-pipelines/onnx/_build/latest?definitionId=7&branchName=main)
-[![Build Status](https://dev.azure.com/onnx-pipelines/onnx/_apis/build/status/MacOS-CI?branchName=main&label=MacOS)](https://dev.azure.com/onnx-pipelines/onnx/_build/latest?definitionId=6&branchName=main)
+[![Build Status](https://dev.azure.com/onnx-pipelines/onnx/_apis/build/status%2FWindows-CI?branchName=refs%2Fpull%2F5568%2Fmerge)](https://dev.azure.com/onnx-pipelines/onnx/_build/latest?definitionId=5&branchName=refs%2Fpull%2F5568%2Fmerge)
```

Comment:
```suggestion
[![Build Status](https://dev.azure.com/onnx-pipelines/onnx/_apis/build/status%2FWindows-CI?branchName=refs%2Fpull%2F5568%2Fmerge)](https://dev.azure.com/onnx-pipelines/onnx/_build/latest?definitionId=5&branchName=refs%2Fpull%2F5568%2Fmerge&label=Windows)
```

try setting the label?

Reply:
should we display results from pulls?

Reply:
> try setting the label?

Good catch. Updated three of them.

> should we display results from pulls? Maybe not?

I think only showing the result from the main branch seems clearer.